example/use_cases/employer_university_flow/employer_university_flow/util.go

```diff
@@ -0,0 +1,315 @@
+package employer_university_flow
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/TBD54566975/ssi-sdk/credential"
+	"github.com/TBD54566975/ssi-sdk/credential/exchange"
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/TBD54566975/ssi-sdk/cryptosuite"
+	"github.com/TBD54566975/ssi-sdk/did"
+	"github.com/TBD54566975/ssi-sdk/util"
+	"github.com/sirupsen/logrus"
+)
+
+var cw = CustomStepWriter{}
```

Comment:
generally don't like mutable state like this...is there another way we can initialize and pass it as the flow goes on?

Reply:
resolved.